src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -413,15 +413,25 @@ public Builder parseArgs(String cmdArgs[])
                     {
                         throw new ConfigurationException("Invalid value of inter_dc_stream_throughput_outbound: " + config.inter_dc_stream_throughput_outbound.toString(), false);
                     }
+
+                    if (config.entire_sstable_stream_throughput_outbound.toMebibytesPerSecond() >= Integer.MAX_VALUE)
+                    {
+                        throw new ConfigurationException("Invalid value of entire_sstable_stream_throughput_outbound: " + config.entire_sstable_stream_throughput_outbound.toString(), false);
```

Comment:
nit: Might be helpful to mention that `config.entire_sstable_stream_throughput_outbound` is too large rather than just invalid?